src/BaselineOfBlocTutorials/BaselineOfBlocTutorials.class.st

```diff
@@ -1,19 +1,19 @@
 Class {
-	#name : #BaselineOfBlocBrickTutorials,
+	#name : #BaselineOfBlocTutorials,
 	#superclass : #BaselineOf,
-	#category : #BaselineOfBlocBrickTutorials
+	#category : #BaselineOfBlocTutorials
 }
 
 { #category : #baselines }
-BaselineOfBlocBrickTutorials >> baseline: spec [
+BaselineOfBlocTutorials >> baseline: spec [
 	<baseline>
 
 	spec for: #common do: [ 
 		spec
 			baseline: 'Memory'
 			with: [ spec repository: 'github://pharo-graphics/Tutorials:dev-1.0/src' ].
 		spec
-			baseline: 'BrickExamplesReordering'
+			baseline: 'BlocReorderingExamples'
```

Comment:
This should have been `'ReorderingExamples' to match the actual name of the baseline class

Reply:
Oh, thanks. I'll push the fix directly to dev-1.0

Reply:
Done. Right now there is a dependency warning that pops up later during load, but you can press Proceed. Until this is fixed.